kod/object/active/holder/nomoveon/battler/player/user.kod

```diff
@@ -3749,6 +3758,41 @@ messages:
       return;
    }
 
+   % Server has received a client request to move an item from the first item's position
```

Comment:
In Blakod function comments should usually be Python-style, i.e. a string right after the function declaration (see AdminGoToOOG in this file for an example).  Then the comment can be seen at runtime in the server.

This comment is kind of wordy; the second sentence doesn't add anything and the first can be shortened.  How about something like "Move the item from payload_index to just before target_index in the user's inventory".

Reply:
Changed format as requested and shortened description.